provisioning/miniprov/hare_mp/systemd.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import re
+# Setup utility for Hare to configure Hare related settings, e.g. logrotate,
+# report unsupported features, etc.
+#
+from typing import List
+
+
+class HaxUnitTransformer:
+    def transform(self, contents: List[str]) -> List[str]:
+        result = []
+        matchers = [
+            r'^[\s]*Requires=',
```

Comment:
We don't want to comment `Requires` and `After` statements but just `Restart` because Consul service is and will not be managed by HA. Also, motr-kernel is not managed by HA presently, until motr-kernel is replaced by libfabric or HA manages it we need to keep the dependency.

Reply:
Alright, I'll fix that. Do we have the list of services that HA doesn't manage at the moment?

Ticket description is quite opaque regarding this question:
```
    Should be disable
    Restart on failure should be removed
    should avoid requires, after for those service which manage by pacemaker.
```

Reply:
Addressed

Reply:
>Do we have the list of services that HA doesn't manage at the moment?

Not a list as such but I have mentioned this at https://jts.seagate.com/browse/EOS-21424?focusedCommentId=2144259&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-2144259